guardian/pluto-start
# Feature/azure compatibility

## What does this change?

Updates to pluto-start to make it work with Azure

## How to test

- Follow the setup instructions at https://github.com/guardian/pluto-headers/pull/8.
- Build and deploy this version of pluto-start
- You should be able to log in via Azure
- Revert the config changes but keep the same software versions
- It should still work against ADFS and Keycloak.

## How can we measure success?

Simple, config-based switchover from ADFS to Azure (and back again, if necessary!)

## Have we considered potential risks?
